QWeb/internal/actions.py

```diff
@@ -136,6 +136,22 @@ def scroll(web_element: WebElement, timeout: int) -> None:  # pylint: disable=un
     javascript.execute_javascript("arguments[0].scrollIntoView();", web_element)
 
 
+def scroll_overlay_adjustment(overlay_height: int = 0) -> None:  # pylint: disable=unused-argument
```

Comment:
`overlay_height` -> `overlay_offset`?
the overlay could be a footer and providing a negative value should take care of the positioning.
Might be kind of annoying to document as scroll direction is not that intuitive (at least for me)

Reply:
Yes, makes sense...I was only thinkin about header. I will check footer, need to generate a test for that too

Reply:
turns out it's close to impossible get element under footer as element is always scrolled to the top. For now I still added support for negative numbers but still pondering if it would be simpler to just support top overlay / positive numbers.

Reply:
I left possibility to have negative offset to helper function in case this is needed in the future. For now current keyword always scrolls the element to top, so it can't be under footer / bottom overlay. Made getcoordinates only approve positive offset values